src/crimson/common/fixed_kv_node_layout.h

```diff
@@ -443,6 +445,22 @@ class FixedKVNodeLayout {
     *layout.template Pointer<0>(buf) = size;
   }
 
+  uint32_t get_checksum() const {
+    return *layout.template Pointer<1>(buf);
+  }
+
+  void set_checksum(uint32_t checksum) {
+    *layout.template Pointer<1>(buf) = checksum;
+  }
+
+  uint32_t calc_crc32c() const {
+    uint32_t crc = ceph_crc32c(
+      1,
+      (unsigned char const *)layout.template Pointer<2>(buf),
+      CAPACITY - layout.template Offset<2>());
+    return ceph_crc32c(crc, (const unsigned char*)buf, sizeof(ceph_le32));
+  }
```

Comment:
CAPACITY is not the total bytes here, right? does the crc include value part?

Reply:
Ah, yes, all fields accept the chksum should be included